pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
-name = "dbt-layer-bigquery"
+name = "dbt-layer"
```

Comment:
Do we want to put a deprecation notice to the old one?

Reply:
Yes, how should we do this?

Reply:
I guess we can do whatever @cihati did with `layer-sdk`, delete all versions and push one special one.